### PR TITLE
NRI cops' file sorting and minor number changes

### PR DESCRIPTION
**Pull request by Stalkeros2**
## About The Pull Request
Finally added some semblance of actual formatting by separating NRI cops-related things in different files.
Also removed the old NRI cops' phonetic alphabet file because, apparently, HL2 Combine soldiers never used phonetic alphabets; and because it was rather rudimentary due to the existance of a normal numbers' folder.

## How This Contributes To The Skyrat Roleplay Experience
Makes it *way* easier to modfiy, search and generally work with the NRI cops' content.
Makes the NRI police officers' codenames more accurate to the inspiration material; removes unneeded file.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
image

</details>

## Changelog

:cl: Stalkeros
del: Phonetic alphabet number global list.
code: Separated NRI police's file into different separate ones.
code: Replace NRI police's phonetic alphabet numbers with the normal ones.
/:cl:

### CODE REVIEW

**Comment by Stalkeros2** [comment]
oops

File: modular_skyrat/modules/encounters/code/nri_cops/ghost_spawners.dm [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+/obj/effect/mob_spawn/ghost_role/human/nri_raider/officer/apply_codename(mob/living/carbon/human/spawned_human)
+	var/callsign = pick(GLOB.callsigns_nri)
+	var/number = pick(GLOB.numbers_as_words)
+	spawned_human.fully_replace_character_name(spawned_human.real_name, "[callsign] [number][GLOB.first_officer == spawned_human ? " Actual" : ""]")
+
+/obj/effect/mob_spawn/ghost_role/human/nri_raider/officer/special(mob/living/carbon/human/spawned_human)
+	. = ..()
+	spawned_human.grant_language(/datum/language/uncommon, TRUE, TRUE, LANGUAGE_SPAWNER)
+	spawned_human.grant_language(/datum/language/yangyu, TRUE, TRUE, LANGUAGE_SPAWNER)
+	spawned_human.grant_language(/datum/language/panslavic, TRUE, TRUE, LANGUAGE_SPAWNER)
```
Comment (vinylspiders) [inline modular_skyrat/modules/encounters/code/nri_cops/ghost_spawners.dm]:
```suggestion
```

You can remove this line. The `. = ..()` means that the base `special()` proc gets called which already grants panslavic to all the subtypes of it. No need to grant it a second time.

File: modular_skyrat/modules/encounters/code/nri_cops/ghost_spawners.dm [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+	faction = list(FACTION_RAIDER)
+	you_are_text = "You are a Novaya Rossiyskaya Imperiya task force."
+	flavour_text = "The station has refused to pay the fine for breaking Imperial regulations, you are here to recover the debt. Do so by demanding the funds. Force approach is usually recommended, but isn't the only method."
+	important_text = "Allowed races are humans, Akulas, IPCs. Follow your field officer's orders. Important mention - while you are listed as the pirates gamewise, you really aren't lore-and-everything-else-wise. Roleplay accordingly."
+	outfit = /datum/outfit/pirate/nri/marine
+	restricted_species = list(/datum/species/human, /datum/species/akula, /datum/species/synthetic)
+	random_appearance = FALSE
+	show_flavor = TRUE
+
+/obj/effect/mob_spawn/ghost_role/human/nri_raider/proc/apply_codename(mob/living/carbon/human/spawned_human)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/encounters/code/nri_cops/ghost_spawners.dm:21]: Document this proc please.

File: modular_skyrat/modules/encounters/code/nri_cops/machinery.dm [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+/obj/machinery/suit_storage_unit/nri
+	mod_type = /obj/item/mod/control/pre_equipped/policing
+	storage_type = /obj/item/tank/internals/oxygen/yellow
+
+/obj/machinery/shuttle_scrambler/nri
+	name = "system crasher"
+	desc = "This heap of machinery locks down supply lines to a halt. Can be turned off, but does not siphon any money. Do that yourself, lazyass."
+	siphon_per_tick = 0
+	var/first_toggle = FALSE
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/encounters/code/nri_cops/machinery.dm:20]: Document this please.

File: modular_skyrat/modules/encounters/code/nri_cops/machinery.dm [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+		first_toggle = TRUE
+	START_PROCESSING(SSobj,src)
+
+/obj/machinery/shuttle_scrambler/nri/process()
+	if(active)
+		if(is_station_level(z))
+			var/datum/bank_account/D = SSeconomy.get_dep_account(ACCOUNT_CAR)
+			if(D)
+				var/siphoned = min(D.account_balance,siphon_per_tick)
+				D.adjust_money(-siphoned)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/encounters/code/nri_cops/machinery.dm]: No single-letter variables please.

File: modular_skyrat/modules/encounters/code/nri_cops/machinery.dm [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+	var/scramble_response = tgui_alert(user, "Turning the crasher on might alienate the population and will make the shuttle trackable by GPS. Are you sure you want to do it?", "Crasher", list("Yes", "Cancel"))
+	if(scramble_response != "Yes")
+		return
+	if(active || !user.can_perform_action(src))
+		return
+	toggle_on(user)
+	update_appearance()
+	send_notification()
+	to_chat(user,span_notice("You toggle [src] [active ? "on":"off"]."))
+	return
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/encounters/code/nri_cops/machinery.dm]: I feel like there's a lot of unnecessary code duplication here, I'm sure you can make this cleaner. Also space out the if statements please.

File: modular_skyrat/modules/encounters/code/nri_cops/machinery.dm [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+
+/obj/machinery/shuttle_scrambler/nri/send_notification()
+	if(active)
+		priority_announce("We're intercepting all of the current and future supply deliveries until you're more cooperative with the dispatch. So, please do be.","NRI IAC HQ",ANNOUNCER_NRI_RAIDERS,"Priority")
+	else
+		priority_announce("We've received a signal to stop the blockade; you're once again free to do whatever you were doing before.","NRI IAC HQ",ANNOUNCER_NRI_RAIDERS,"Priority")
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/encounters/code/nri_cops/machinery.dm]:
```suggestion
	priority_announce(active ?
		"We're intercepting all of the current and future supply deliveries until you're more cooperative with the dispatch. So, please do be." :
		"We've received a signal to stop the blockade; you're once again free to do whatever you were doing before.",
		"NRI IAC HQ",
		ANNOUNCER_NRI_RAIDERS,
		"Priority"
	)
```

File: modular_skyrat/modules/encounters/code/nri_cops/outfits.dm [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+	uniform = /obj/item/clothing/under/costume/nri
+	suit = null
+
+	gloves = /obj/item/clothing/gloves/combat
+
+	shoes = /obj/item/clothing/shoes/combat
+
+	belt = /obj/item/storage/belt/security/nri
+	back = /obj/item/storage/backpack/satchel/leather
+	backpack_contents = list(/obj/item/storage/box/nri_survival_pack/raider = 1, /obj/item/crucifix = 1, /obj/item/ammo_box/magazine/m9mm = 3, /obj/item/clothing/mask/gas/hecu2 = 1, /obj/item/modular_computer/pda/security = 1)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/encounters/code/nri_cops/outfits.dm]:
```suggestion
	backpack_contents = list(
		/obj/item/storage/box/nri_survival_pack/raider = 1,
		/obj/item/crucifix = 1,
		/obj/item/ammo_box/magazine/m9mm = 3,
		/obj/item/clothing/mask/gas/hecu2 = 1,
		/obj/item/modular_computer/pda/security = 1,
	)
```

File: modular_skyrat/modules/encounters/code/nri_cops/outfits.dm [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+	uniform = /obj/item/clothing/under/costume/nri/captain
+	suit = null
+
+	gloves = /obj/item/clothing/gloves/combat
+
+	shoes = /obj/item/clothing/shoes/combat
+
+	belt = /obj/item/storage/belt/security/nri
+	back = /obj/item/storage/backpack/satchel/leather
+	backpack_contents = list(/obj/item/storage/box/nri_survival_pack/raider = 1, /obj/item/ammo_box/magazine/m9mm_aps = 3, /obj/item/gun/ballistic/automatic/pistol/ladon/nri = 1, /obj/item/crucifix = 1, /obj/item/clothing/mask/gas/hecu2 = 1, /obj/item/modular_computer/pda/security = 1)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/encounters/code/nri_cops/outfits.dm]:
```suggestion
	backpack_contents = list(
		/obj/item/storage/box/nri_survival_pack/raider = 1,
		/obj/item/ammo_box/magazine/m9mm_aps = 3,
		obj/item/gun/ballistic/automatic/pistol/ladon/nri = 1,
		/obj/item/crucifix = 1, /obj/item/clothing/mask/gas/hecu2 = 1,
		/obj/item/modular_computer/pda/security = 1,
	)
```

File: modular_skyrat/modules/encounters/code/nri_cops/shuttle.dm [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+	name = "NRI Starship"
+	forced_ambience = TRUE
+	ambient_buzz = 'modular_skyrat/modules/encounters/sounds/amb_ship_01.ogg'
+	ambient_buzz_vol = 15
+	ambientsounds = list('modular_skyrat/modules/encounters/sounds/alarm_radio.ogg',
+						'modular_skyrat/modules/encounters/sounds/alarm_small_09.ogg',
+						'modular_skyrat/modules/encounters/sounds/gear_loop.ogg',
+						'modular_skyrat/modules/encounters/sounds/gear_start.ogg',
+						'modular_skyrat/modules/encounters/sounds/gear_stop.ogg',
+						'modular_skyrat/modules/encounters/sounds/intercom_loop.ogg')
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/encounters/code/nri_cops/shuttle.dm]:
```suggestion
	ambientsounds = list(
		'modular_skyrat/modules/encounters/sounds/alarm_radio.ogg',
		'modular_skyrat/modules/encounters/sounds/alarm_small_09.ogg',
		'modular_skyrat/modules/encounters/sounds/gear_loop.ogg',
		'modular_skyrat/modules/encounters/sounds/gear_start.ogg',
		'modular_skyrat/modules/encounters/sounds/gear_stop.ogg',
		'modular_skyrat/modules/encounters/sounds/intercom_loop.ogg',
	)
```

File: modular_skyrat/modules/encounters/code/nri_cops/storage.dm [hunk 10]
```diff
@@ -0,0 +1,25 @@
+/obj/item/storage/belt/military/nri/captain/pirate_officer/PopulateContents()
+	generate_items_inside(list(
+		/obj/item/knife/combat = 1,
+		/obj/item/grenade/smokebomb = 1,
+		/obj/item/grenade/flashbang = 1,
+	),src)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/encounters/code/nri_cops/storage.dm]:
```suggestion
		),
		src,
	)
```

File: modular_skyrat/modules/encounters/code/nri_cops/storage.dm [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+
+/obj/item/storage/belt/security/nri/PopulateContents()
+	generate_items_inside(list(
+		/obj/item/knife/combat = 1,
+		/obj/item/grenade/smokebomb = 1,
+		/obj/item/grenade/flashbang = 1,
+	),src)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/encounters/code/nri_cops/storage.dm]:
```suggestion
		),
		src,
	)
```

File: modular_skyrat/modules/encounters/code/nri_cops/turret.dm [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+	lethal_projectile_sound = 'modular_skyrat/modules/encounters/sounds/shell_out_tiny.ogg'
+	stun_projectile_sound = 'modular_skyrat/modules/encounters/sounds/shell_out_tiny.ogg'
+
+/obj/machinery/porta_turret/syndicate/nri_raider/target(atom/movable/target)
+	if(target)
+		setDir(get_dir(base, target))//even if you can't shoot, follow the target
+		shootAt(target)
+		addtimer(CALLBACK(src, PROC_REF(shootAt), target), 4)
+		addtimer(CALLBACK(src, PROC_REF(shootAt), target), 8)
+		addtimer(CALLBACK(src, PROC_REF(shootAt), target), 12)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/encounters/code/nri_cops/turret.dm]: I feel like those numbers should've been defines at the start of the file (and undefined at the end of it).

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Review by vinylspiders — CHANGES_REQUESTED** [review]

**Comment by Stalkeros2** [comment]
Gonna check all the reviews somewhere tomorrow (and work on them a day later) as I was too busy today.

File: modular_skyrat/modules/encounters/code/nri_cops/turret.dm (hunk 12 above)
Comment (Stalkeros2) [reply modular_skyrat/modules/encounters/code/nri_cops/turret.dm]: They're used once; why would I.

File: modular_skyrat/modules/encounters/code/nri_cops/machinery.dm (hunk 5 above)
Comment (Stalkeros2) [reply modular_skyrat/modules/encounters/code/nri_cops/machinery.dm]: I literally can't code; I have no idea how to make it better until maybe like a week after this ever gets merged and a sudden revelation that'll come to me in a dream; even joke aside, this is probably the best solution I might come up with, as it's already reasonably good. Excluding the code spacing which's easy as hell and was done a long time ago but I think I forgot to push commits.

**Comment by Stalkeros2** [comment]
I'll redo this at a slightly later date, when I'm done with a bunch of removals.

**Pull request closed without merging**